### PR TITLE
add finance prolicy about proposals being public

### PR DESCRIPTION
**Pull request by eteq**
Just what it says on the tin!

### CODE REVIEW

**Comment by pllim** [comment]
Title: prolicy -> policy

File: finance/finance-ctte-policy.rst [hunk 1]
```diff
@@ -84,6 +84,13 @@ Privacy and confidentiality
 
    __ https://docs.google.com/document/d/193ERJ6wMNPaNjGSWbS94KHGnojpZ_zwXxWTi9L2etSo/edit#bookmark=id.bwbxtdjn2
 
+
+#. Funding proposals that are written on behalf of the Astropy Project should be public and open for comment to the Astropy community unless there is a specific reason they cannot be.  Similarly, progress reports on such grants required by the provider of the grant should also be shared publicly with the Astropy community if at all possible.
+
+   (Agreed `2020-10-2`__.)
```
Comment (astrojuanlu) [inline finance/finance-ctte-policy.rst:90]:
```suggestion
   (Agreed `2020-10-02`__.)
```

File: finance/finance-ctte-policy.rst [hunk 2]
```diff
@@ -84,6 +84,13 @@ Privacy and confidentiality
 
    __ https://docs.google.com/document/d/193ERJ6wMNPaNjGSWbS94KHGnojpZ_zwXxWTi9L2etSo/edit#bookmark=id.bwbxtdjn2
 
+
+#. Funding proposals that are written on behalf of the Astropy Project should be public and open for comment to the Astropy community unless there is a specific reason they cannot be.  Similarly, progress reports on such grants required by the provider of the grant should also be shared publicly with the Astropy community if at all possible.
```
Comment (jdswinbank) [inline finance/finance-ctte-policy.rst:88]: One sentence per line, for consistency with the rest of the document?

Comment (jdswinbank) [inline finance/finance-ctte-policy.rst:88]:
```suggestion
Funding proposals that are written on behalf of the Astropy Project should be public, and comments should be solicited from the community before the proposal is submitted, unless there are specific reasons why this is not possible.
```

**Review by jdswinbank — APPROVED** [review]

**Comment by jdswinbank** [comment]
Hey @eteq — I just noticed this PR has been sitting idle for a while now. Anything I can do to help get it merged?

**Review by hamogu — APPROVED** [review]

**Comment by hamogu** [comment]
We've discussed this before, agreed on it months ago and it's been open for review for quite a while. MErging.

**Pull request merged**